IPython/core/usage.py

```diff
@@ -76,6 +76,22 @@
 At your system command line, type 'ipython -h' to see the command line
 options available. This document only describes interactive features.
 
+GETTING HELP
+------------
+
+Within IPython you have various way to access help:
+
+  ?         -> Introduction and overview of IPython's features (this screen).
+
+  %quickref -> Quick reference of all IPython specif syntax and magics.
+
+  object?   -> Details about 'object', use 'object??' for extra details.
```

Comment:
I would move this one above so that '?' and 'foo?' are discussed next to each other.